web/satellite/src/types/analytics.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.vue' {
```

Comment:
We should only have one defined interface for segment

Reply:
hmm don't remember adding this, removed!